apps/mobile-mzima-client/src/app/core/services/language.service.ts

```diff
@@ -0,0 +1,66 @@
+import { Injectable } from '@angular/core';
```

Comment:
These files are more or less copied from the web-client. I think we should be able to move it to the sdk once we have optimised the translation-files and removed duplicates etc so we can use the same files for mobile and web.